### PR TITLE
xGov-19 - Proposal to monitor the Algorand Foundation market operations

### PR DESCRIPTION
**Pull request by papasotiriou**
Independent audit

### CODE REVIEW

File: Proposals/xgov-19.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Algorand Foundation market operations dashboard
+author: Sot Papasot (@papasotiriou)
+company_name: datascientist.algo
+category: Tools
+focus_area: Monitoring
+open_source: Yes
+amount_requested: 4000
+status: Draft
```
[the remark this answers is not in the document]
Comment (papasotiriou) [reply Proposals/xgov-19.md:9]: Sure, thanks for pointing that out

**Comment by ShaperOfEntropy** [comment]
I like the idea of independent, real-time auditing of AF operations compared to just their quarterly reports.
The amount requested seems reasonable for the effort required.
My comments:
1. I'm concerned how will it be maintained long-term. Will additional funding be required by xGovs to keep the product up and running? Is it indented to be supported based on donations?
2. In the proposals it is described what kind of visualizations have been done already and not specifically which ones will be included additionally.
3. How will the grant be spent? Will ALGO be sold on the market? If yes, using DEXs or CEXs? Or will a loan be taken against it in DeFi/CeFi? Would you be willing to vest the awarded grant in a smart contract?

**Comment by papasotiriou** [comment]
I'll try to answer your comments very briefly, so feel free to discuss if you would like to hear more from me. 

1. I am using a third party data source which means that the dashboard will be online as long as [GeniiData](https://geniidata.com/) supports Algorand. I have first published it 5 months ago, doing minor modifications to improve it based on user feedback. I'll keep the product in its current form regardless of the xgov proposal outcome without any other donations. 
2. Ideally, I'd like to track all of the Algorand foundation activities in similar format, offering multiple pages. This is like a demo to illustrate what can be done by the community to help with transparency and decentralisation. Therefore, the Algo amount requested is for the time I spend creating this product with the current visualisations. Obviously, I tried to convert my hourly work rate to Algo, which is very challenging given its price action.  
3. Yes, I am happy with a vesting period smart contract, even though I have no idea how to develop it. For example, if the product is up and running with accurate data by the end of the year, then I receive the award. I'll hold Algo in Defi governance or swap them for a more promising Algorand standard asset.

**Comment by ShaperOfEntropy** [comment]
Thanks for your clarifications!
I'm happy to see your grant request has already been approved.
To further clarify on the first point of maintainability, is [GeniiData](https://geniidata.com/) free to use? Does it have any limitations (e.g. max number of requests)? What if it were to cease operation? How difficult would it be to restart the service on another platform?

**Comment by papasotiriou** [comment]
GeniiData is free to use, but it also offers premium features such as downloading the data. I think they will keep it like that because dashboards are promoting their platform. 

The most important limitation is that the graphs and tables are not always real time and sometimes they could be lagging for a few hours. 

There are alternatives like [bitquery](https://ide.bitquery.io/), but it will take some time to rewrite the queries and create the visualizations.

**Comment by nonfungibleab** [comment]
Congratulations! xGov voting session 1 is now closed and your proposal has passed. I need your email to kick off contract signature next week. Would you please send me a DM on Twitter or Discord?

**Pull request merged**